www/src/app/views/InboxContainer.js

```diff
@@ -223,6 +219,22 @@ export default class InboxContainer extends Component {
         handleResolve={this.handleResolve}
         handleArchive={this.handleArchive}
       />
+      </section>
     );
   }
 }
+
+const mapStateToProps = (state) => {
+  return {
+    author: state.user.author,
+    updates: state.updates.inbox
```

Comment:
definitely interested in discussing your vision for the store model here! I think I don't fully grasp the functionality of different items, and how you want to structure them - let's see if we can arrange a time to chat about it.